NitroxPatcher/Patches/Dynamic/uGUI_SignInput_OnDeselect_Patch.cs

```diff
@@ -1,12 +1,12 @@
 ﻿using System;
 using System.Reflection;
 using Harmony;
-using NitroxModel.Core;
 using NitroxClient.GameLogic;
+using NitroxModel.Core;
 
-namespace NitroxPatcher.Patches
+namespace NitroxPatcher.Patches.Dynamic
 {
-    public class uGUI_SignInput_OnDeselect_Patch : NitroxPatch
+    public class uGUI_SignInput_OnDeselect_Patch : NitroxPatch, IDynamicPatch
```

Comment:
Do we still need NitroxPatch now that we have INitroxPatch?

Reply:
I could have used NitroxPatch directly instead of INitroxPatch. Though I liked INitroxPatch with just the `Patch` and `Restore` methods on it because that's all the `NitroxPatcher.Execute` needed.